appdev/guides/index.rst

```diff
@@ -3,22 +3,44 @@
 Developer guides
 ================
 
-Get started building your first app or learn about advanced concepts with our developer guides.
+To get started with your first app check out the following developer guides:
+
+* `Ubuntu Touch programming book <https://mimecar.gitbooks.io/ubuntu-touch-programming-course/content/>`__
```

Comment:
Maybe also add:

[Qt doc: First Steps with QML](https://doc.qt.io/qt-5/qmlfirststeps.html)
[Qt doc: Qt Examples And Tutorials](https://doc.qt.io/qt-5.12/qtexamplesandtutorials.html)
[Qt docs: All 5.12 QML Types](https://doc.qt.io/qt-5.12/qmltypes.html)